src/index.css

```diff
@@ -95,7 +92,7 @@ a {
   line-height: normal;
   font-weight: bold;
   letter-spacing: 1.9px;
-  color: #240b0b;
+  color: #1c1111;
 }
```

Comment:
- Great Job for modifying the style as the previous code reviewer suggested you :+1: I think you need to do a little bit to make the font, bg, and colors your, you can use [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378/stage) inspection :+1: 
here is your design
![Screenshot from 2023-03-02 18-03-10](https://user-images.githubusercontent.com/52010789/222468327-26464aae-7361-4539-a4b7-b6a63035e742.png)
here is zipline design
![Screenshot from 2023-03-02 18-04-13](https://user-images.githubusercontent.com/52010789/222468697-32266ac5-a980-46cf-aa19-baea411fe53a.png)